test/eventlistener_test.go

```diff
@@ -36,7 +36,7 @@ func TestEventListenerCreate(t *testing.T) {
 	t.Log("Start EventListener e2e test")
 
 	// Create EventListener
-	el, err := c.TriggersClient.TriggersV1alpha1().EventListeners(namespace).Create(
+	el, err := c.TriggersClient.TektonV1alpha1().EventListeners(namespace).Create(
```

Comment:
This file has a few `t.Logf` that do not use formatting. They were already there prior this commit, but they could be cleaned up.

Reply:
Good catch, I fixed them 👍